tests/rpc/rpc_test.go

```diff
@@ -565,7 +565,7 @@ func TestEth_EthResend(t *testing.T) {
 func TestEth_FeeHistory(t *testing.T) {
 	params := make([]interface{}, 0)
 	params = append(params, 4)
-	params = append(params, "0x1c")
+	params = append(params, "0xa")
```

Comment:
slightly modify to a lower value in case the chain is running behind or the integration tests are running too fast